compare_rpm_repo.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+
+import argparse
+import bz2
+import json
+import lzma
+import requests
+import sqlite3
+import subprocess
+import tempfile
+import xml.etree.ElementTree as ET
+from collections import defaultdict
+from pathlib import Path
+
+from packaging.version import Version
+from list_updatable_packages import Spec
```

Comment:
this implies a `git mv list_updatable_packages list_updatable_packages.py` but I didn't include that in this PR as it also would require changes to
https://github.com/theforeman/foreman-packaging/blob/e4e1ab5b0b73559ff41c4ca48fb61680db992bb7/.github/workflows/bump_packages.yml#L26
and I wonder if there is a better way

Reply:
I thought about merging it into a single script and make it something like `./util --list-updatable` vs `./util --compare-git-and-repo`, but given the size of this script and the other one I think it could become too big.